content/300-guides/050-database/880-supabase.mdx

```diff
@@ -28,6 +28,28 @@ Many aspects of using Prisma with Supabase are just like using Prisma with any o
 - use [`db push`](/concepts/components/prisma-migrate/db-push) to push changes in your schema to Supabase
 - use [Prisma Client](/concepts/components/prisma-client) in your application to talk to the database server at Supabase
 
+## Specific considerations
+
+If your application is being deployed to a serverless environment, make sure that you follow our general guidance on [connection pooling](https://www.prisma.io/docs/guides/performance-and-optimization/connection-management#recommended-connection-pool-size-1). Supabase makes this easy by [providing connection pooling](https://supabase.com/docs/guides/integrations/prisma#connection-pooling-with-supabase) compatible with PgBouncer.
```

Comment:
The specific link for "connection pooling" you chose is unfortunately terrible, as it is just invalid guidance we have in our docs. It makes no sense to use a connection pool size of 1 if you have a pooler before your database that prevents you running out of connections.

Reply:
Ah okay, bummer. I can change this to be more specific to Supabase and say that a connection pooler should be used. In this case, Supabase's PgBouncer implementation